source/xep/httpFileUpload/DownloadManager.cpp

```diff
@@ -86,73 +86,24 @@ void DownloadManager::doDownload(const QUrl &requestedUrl)
     {
         QNetworkRequest request(url);
         QNetworkReply *reply = manager.get(request);
+        FileWithCypher *file = new FileWithCypher(pathAndFile, this);
 
-        reply->setProperty(kIvAndKey, requestedUrl.fragment());
-        reply->setProperty(kPathAndFile, pathAndFile);
-        reply->setProperty(kOriginalUrl, requestedUrl);
+        if (!file->open(QIODevice::WriteOnly))
+        {
+            fprintf(stderr, "Could not open %s for writing: %s\n",
+                    qPrintable(file->fileName()),
+                    qPrintable(file->errorString()));
```

Comment:
Dont continue the program flow if the file could not be opened for writing.

Reply:
ok